qiskit/providers/aer/noise/errors/standard_errors.py

```diff
@@ -191,7 +191,7 @@ def single_pauli(pauli):
             # Pauli strings go from qubit-0 on left to qubit-N on right
             # but pauli ops are tensor product of qubit-N on left to qubit-0 on right
             # We also drop identity operators to reduce dimension of matrix multiplication
-            mat = 1
+            mat = np.identity(1)
```

Comment:
This would be better as `mat = None` and then later checking if `if mat is None` for the identity case.

Reply:
Actually this is wrong. I forgot that the init `mat` was used in `np.kron` below